system-tests/projects/runner-e2e-specs/cypress/e2e/errors/route.cy.js

```diff
@@ -1,4 +1,19 @@
-import { abortXhr, sendXhr } from './setup'
+// eslint-disable-next-line
```

Comment:
what is eslint complaining about here? The export? Why do we need to export these from the spec?

Reply:
I think this (and the exports) are just some copy-paste holdovers, I'll get this cleaned up.